integration/test/draining/draining_test.go

```diff
@@ -231,7 +231,7 @@ func Test_Draining(t *testing.T) {
 }
 
 func numberOfMasters(clusterID string) (int, error) {
-	cluster, err := h.AWSCluster(clusterID)
+	cluster, err := config.Host.AWSCluster(clusterID)
```

Comment:
Just to keep this in mind. The wording is still off here. Should probably be `Tenant` and `ControlPlane`.